### ISSUE TITLE
Docker Build Fail

### ISSUE THREAD

**Issue by cqllang** [EOSIO/eos#1146]
When I run
`docker build . -t eosio/eos`

Reported the following error
`Step 20/27 : COPY --from=builder /opt/eos/bin /opt/eos/bin
COPY failed: stat /var/lib/docker/overlay2/d66370d8a0140a0596ce407855eb13790db1a051570b4771bad485381c8853be/merged/opt/eos/bin: no such file or directory`

System: Ubuntu 16.04.3 LTS
Docker version 17.09.1-ce, build 19e2cf6

How to solve it?

**Comment by kryptokah** [comment]
Same issue for me. I did a clean pull (commit 22acff0c733e1c9eb74d35873fc9e6ca28cab2b7), and ran  `docker build . -t eosio/eos`. Things failed at this point:

```
Step 20/27 : COPY --from=builder /opt/eos/bin /opt/eos/bin
COPY failed: stat /var/lib/docker/aufs/mnt/96ecbe8bd8c23bf5a3b8486b37e25871f9d24b5d656d5499fdc90a2fadc58c32/opt/eos/bin: no such file or directory
```

System: Ubuntu 16.04.1 LTS
Docker version 17.12.0-ce, build c97c6d6

**Comment by Amazing110** [comment]
same problem while building docker image, anyboby help pls?

**Comment by manxiaqu** [comment]
same problem.  
`FROM ubuntu:16.04 as builder`
`# FROM huangminghuang/eos_builder as builder`
`COPY --from=builder /opt/eos/bin /opt/eos/bin `
`COPY --from=builder /tmp/build/contracts /contracts `
`COPY --from=builder /eos/Docker/config.ini /eos/genesis.json`
we can know that the reason of error is that the builder(ubuntu:16.04) don't contain folder /opt/eos.
Also check huangminghuang/eos_builder, but only find start_eosd.sh and data-dir in /opt/eos/bin.
is there any wrong with it?

**Comment by atjason** [comment]
Same problem.

System: macOS High Sierra 10.13.2 (17C88)
Docker version Docker version 17.12.0-ce, build c97c6d6

**Comment by GabrielNicolasAvellaneda** [comment]
Hey guys, I made a change to make it work. Please take a look at this PR https://github.com/EOSIO/eos/pull/1184

**Comment by atjason** [comment]
@GabrielNicolasAvellaneda it works, thanks a lot.

**Comment by GabrielNicolasAvellaneda** [comment]
Glad to hear that. Please don't close this until the PR is merged or closed. Thanks!

**Comment by myenglishmail** [comment]
can any one upload the compiled binary docker image that I can download it ?

**Comment by ericiles** [comment]
This has been resolved with PR #1218
